persist/sqlite/accounts.go

```diff
@@ -265,7 +460,7 @@ func updateContractUsage(tx *txn, accountID int64, usage accounts.Usage, log *za
 			return fmt.Errorf("failed to decrement account funding: %w", err)
 		}
 
-		if contract.Status == contracts.ContractStatusActive || contract.Status == contracts.ContractStatusPending {
+		if contract.Status == contracts.ContractStatusActive {
```

Comment:
What triggered this change? Isn't this pretty much guaranteed to miss metrics since we start using accounts before the contract appears on chain?

Reply:
The contract metrics are added to the global metrics when the status changes from pending -> active, which is consistent with the contract revision PR that just merged. This was a necessary change when finalization was a thing. Now that it's not we could revert it, but it does simplify the edge-cases around pending->reverted->confirmed.